.env-dist

```diff
@@ -166,3 +166,6 @@ NIMBUS_SIDECAR_URL=http://localhost:8001
 
 # The maximum number of jobs that the email breach alert worker will process.
 EMAIL_BREACH_ALERT_MAX_MESSAGES = 10000
+
+NEXT_PUBLIC_LEARN_MORE_ABOUT_MONITOR_PLUS_URL=https://support.mozilla.org/kb/how-does-monitor-plus-work
+NEXT_PUBLIC_FAQ_MONITOR=https://support.mozilla.org/en-US/kb/firefox-monitor-faq
```

Comment:
```suggestion
NEXT_PUBLIC_FAQ_MONITOR=https://support.mozilla.org/kb/firefox-monitor-faq
```

Reply:
Semi-related (but probably outside the scope of this PR), but it looks like we have a couple other `/en-US/` references in `/src/app/(proper_react)/redesign/(public)/LandingView.tsx`

```sh
git grep -in "/en-us/" -- ":(exclude)*.md" | grep -Ev "\s+(//|\*)\s+"

.env-dist:171:NEXT_PUBLIC_FAQ_MONITOR=https://support.mozilla.org/en-US/kb/firefox-monitor-faq
src/app/(proper_react)/redesign/(public)/LandingView.tsx:97:                            href="https://www.mozilla.org/en-US/firefox/privacy/"
src/app/(proper_react)/redesign/(public)/LandingView.tsx:111:                            href="https://www.mozilla.org/en-US/firefox/privacy/"
```

https://github.com/mozilla/blurts-server/blob/c5f95f8c0b6967fe418ba16801c7f16cd3311ba1/src/app/(proper_react)/redesign/(public)/LandingView.tsx#L95-L99

https://github.com/mozilla/blurts-server/blob/c5f95f8c0b6967fe418ba16801c7f16cd3311ba1/src/app/(proper_react)/redesign/(public)/LandingView.tsx#L109-L113